packages/app/src/runner/ResizablePanels.vue

```diff
@@ -72,6 +84,7 @@ import type { DraggablePanel } from './useRunnerStyle'
 const props = withDefaults(defineProps<{
   showPanel1?: boolean // specsList in runner
   showPanel2?: boolean // reporter in runner
```

Comment:
what is panel 3?

Reply:
Panel 3 is the AUT and is always displayed.